### PR TITLE
Navigation TopBar

### PR DESCRIPTION
**Pull request by sfernandez11**
# Summary
## Features
* Add navigation top bar for navigation between views.
* Add sign out.
* Changes in views styles.
## Screenshots
image
image
## Trello Card
https://trello.com/c/2uUPW8f1/25-routing-ii-topbar

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -33,17 +36,39 @@ PrivateRoute.propTypes = {
   location: PropTypes.string
 };
 
+const TopBar = () => (
+  <div className="topBar">
+    <Link className="gameLink" to="/game">
+      Game
```
Comment (IgnacioAbadie) [inline src/index.js]: remember extract the strings

File: src/index.js [hunk 2]
```diff
@@ -33,17 +36,39 @@ PrivateRoute.propTypes = {
...
-        </Switch>
-      </Router>
-    </Fragment>
+    <Router history={history}>
+      <Switch>
+        <Route exact path="/" render={() => <Redirect to="/login" />} />
+        <Route path="/login" component={LogIn} />
+        <PrivateRoute path="/game" component={Game} />
+        <PrivateRoute path="/profile" component={Empty} />
+        <Route
```
Comment (IgnacioAbadie) [inline src/index.js]: dont need a route to do the logout, just use the sign out button and dispatch an action that remove token and push login view

**Review by IgnacioAbadie — CHANGES_REQUESTED** [review]
Just 2 little changes

File: src/index.js (hunk 1 above)
Comment (sfernandez11) [reply src/index.js]: Done

File: src/index.js (hunk 2 above)
Comment (sfernandez11) [reply src/index.js]: Done

File: src/app/components/TopBar/topBar.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import './styles.css';
+
+class TopBar extends Component {
+  handleClick = () => {
+    this.props.dispatch(actionCreators.signOut());
+  };
+
+  render() {
+    return (
+      <div className="topBar">
```
Comment (IgnacioAbadie) [inline src/app/components/TopBar/topBar.js]: check classnames should be like `my-class-name` convention.

**Review by IgnacioAbadie — CHANGES_REQUESTED** [review]
1 comment and go

File: src/app/components/TopBar/topBar.js (hunk 3 above)
Comment (sfernandez11) [reply src/app/components/TopBar/topBar.js]: Done

**Review by IgnacioAbadie — APPROVED** [review]

**Pull request merged**